e2e/pages/app/marquee-wizard/marquee-wizard.testpage.component.html

```diff
@@ -29,6 +29,7 @@
 
 <button id="resizeable" class="btn button-primary" (click)="resizable = true">Enable resizable</button>
 <button id="footerTemplate" class="btn button-secondary" (click)="hasFooterTemplate = true">Add footerTemplate</button>
+<button id="validation" class="btn button-secondary" (click)="validation = true">Add Validation</button>
```

Comment:
"validation" implies that it is enabling checking to see if it is valid or not rather than marking the step as invalid. Could this be changed to something like "Set Step Invalid" and perhaps perhaps we could flip the variable to be `invalid` or `stepInvalid` to make it clear.